caffe2/CMakeLists.txt

```diff
@@ -400,9 +400,6 @@ if (BUILD_TEST)
     target_link_libraries(${test_name} ${Caffe2_MAIN_LIBS} gtest_main)
     target_include_directories(${test_name} PRIVATE $<INSTALL_INTERFACE:include>)
     target_include_directories(${test_name} PRIVATE ${Caffe2_CPU_INCLUDE})
-    if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.0)
-      target_compile_features(${test_name} PRIVATE cxx_range_for)
```

Comment:
Old version of cmake (e.g. 3.5) doesn't know about clang 7 and so it doesn't have the list of supported compiler features of it. On the other hand, this seems to be legacy way of specifying c++11 in cmake pre 3.1 world (since which cmake has introduced `CMAKE_CXX_STANDARD` and `CMAKE_C_STANDARD`.